README.md

```diff
@@ -37,6 +37,9 @@ pip install cloud-sql-python-connector[driver]
 ```
 For example, to use the Python connector with `pymysql`, run `pip install cloud-sql-python-connector[pymysql]`
 
+**Note:** MacOS users with the **zsh** terminal, please use
```

Comment:
This is a problem in zsh on Linux as well. I think it's probably easiest to just put quotes around the library name for all users.

Reply:
That fails for Windows command prompt then. I thought of that too but trying it on Windows didn't work.

Reply:
Does double quotes work on windows `pip install "cloud-sql-python-connector[pymysql]"?

Reply:
Yup double quotes actually does seem to work on Windows

Reply:
Let's update to double quotes in the example then.

Also, would it be worth just listing all the drivers and their corresponding pip command?